### PR TITLE
Rust relative datetime testing

### PR DESCRIPTION
**Pull request by sven-oly**
Now compiling!

### CODE REVIEW

**Comment by sven-oly** [comment]
Having problems compiling with DataLocale.

Also not sure how to select the correct constructor based on style option and unit.

**Comment by sven-oly** [comment]
This is ready for review. The failures and unsupported resultsare due to:
a. "und" locale
b. numbering system not implemented for the numeric output
PTAL

**Comment by sven-oly** [comment]
Ready for review. Test failures are from:
a. non-Latn numbering system
b. "und" locale

File: executors/rust/1.3/Cargo.toml [hunk 1]
```diff
@@ -8,12 +8,14 @@ edition = "2021"
 log = "0.4"
 env_logger = "0.9.1"    
 
+icu_provider = "1.3.2"
```
Comment (sffc) [inline executors/rust/1.3/Cargo.toml:11]:
```suggestion
icu_provider = "~1.3"
```

File: executors/rust/1.3/src/relativedatetime_fmt.rs [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+    let lang_id = if let Ok(lc) = locale_str.parse::<Locale>() {
+        lc
+    } else {
+        return Ok(json!({
+            "label": label,
+            "error_detail": {"locale": locale_str},
+            "error_type": "locale problem",
+        }));
+    };
+    let data_locale = DataLocale::from(lang_id);
```
[the remark this answers is not in the document]
Comment (sffc) [reply executors/rust/1.3/src/relativedatetime_fmt.rs:147]: It looks like this is done

**Review by sffc — APPROVED** [review]

**Review by sffc — APPROVED** [review]

**Pull request merged**